examples/standalone/02_hello_window/src/main.rs

```diff
@@ -0,0 +1,169 @@
+use std::sync::Arc;
+
+use winit::application::ApplicationHandler;
+use winit::event::WindowEvent;
+use winit::event_loop::{ActiveEventLoop, ControlFlow, EventLoop};
+use winit::window::{Window, WindowId};
+
+struct State {
+    surface: wgpu::Surface<'static>,
+    device: wgpu::Device,
+    adapter: wgpu::Adapter,
+    queue: wgpu::Queue,
+    size: winit::dpi::PhysicalSize<u32>,
+}
+
+impl State {
+    // Creating some of the wgpu types requires async code
+    async fn new(window: Arc<Window>) -> State {
+        let size = window.inner_size();
+
+        // The instance is a handle to our GPU
+        // Backends::all => Vulkan + Metal + DX12 + Browser WebGPU
+        let instance = wgpu::Instance::new(&wgpu::InstanceDescriptor {
+            backends: wgpu::Backends::PRIMARY,
+            ..Default::default()
+        });
```

Comment:
We should be using all backends here, and this could really be `wgpu::InstanceDescriptor::default()`